oyaml.py

```diff
@@ -18,6 +18,7 @@ def map_constructor(loader, node):
 
 pyyaml.add_representer(dict, map_representer)
 pyyaml.add_representer(OrderedDict, map_representer)
+pyyaml.add_representer(OrderedDict, map_representer, Dumper=pyyaml.dumper.SafeDumper)
```

Comment:
Interesting, why are both of the above lines needed?

Reply:
`pyyaml.add_representer` works on the `Dumper` class obj passed into it.

Reply:
Okay, but if I remove line 20 everything still works for me. Do you have a test that fails when that line is removed?

Reply:
Really?  That's odd.  For me, the existing `test_dump` and `test_dump_all` fail without line 20.  Could you send me your stacktrace and pip freeze.

Reply:
ah, "works for me" is not "running the oyaml tests" :-) All good, if you have tests that prove it's needed, that's weird, but okay :-)

Reply:
OK.  I've released v0.3 to warehouse (the new PyPI).

Reply:
git tags are a thing you know ;-)